<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Leds_widget</class>
 <widget class="QWidget" name="Leds_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_leds">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame_leds{
	background:rgb(20,20,20);
}</string>
   </property>
   <widget class="QFrame" name="frameLedsBulb">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:125px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="ledsBulb">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>250</width>
       <height>250</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:black;
</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ledsOnOffButton">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>95</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiBold Condensed</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{
	border-radius:30px;
	

}
QCheckBox:checked{
	
}
QCheckBox::indicator {
	width:60px;
	height:60px;
	background:rgba(0, 0, 0, 170);
	border-radius:30px;
}
QCheckBox::indicator:checked {
		background:rgba(0, 0, 0, 0);
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>95</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiBold SemiConden</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
border: 2px solid white;
border-radius:30px;
background-color: rgba(0, 0, 0, 100);</string>
     </property>
     <property name="text">
      <string>I</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>ledsBulb</zorder>
    <zorder>label_5</zorder>
    <zorder>ledsOnOffButton</zorder>
   </widget>
   <widget class="QFrame" name="frameLedButtons">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>700</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:rgb(94, 94, 94);
	border:2px solid white;
	border-radius:20px;
	color:white;

}</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_8">
       <widget class="QPushButton" name="ledsProgramJumpButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>124</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift Light</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	
	background-color: rgb(49, 49, 49);
	border: 2px solid rgb(181, 0, 0);
	border-radius:20px;
	color:gray;
}
QPushButton:pressed{
	
	
	background-color: rgb(188, 0, 0);
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Jump</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_7">
       <widget class="QPushButton" name="ledsProgramFadeButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>124</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift Light</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	
	background-color: rgb(49, 49, 49);
	border: 2px solid rgb(181, 0, 0);
	border-radius:20px;
	color:gray;
}
QPushButton:pressed{
	
	
	background-color: rgb(188, 0, 0);
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Fade</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_6">
       <widget class="QPushButton" name="ledsProgramRandomButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>124</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift Light</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	
	background-color: rgb(49, 49, 49);
	border: 2px solid rgb(181, 0, 0);
	border-radius:20px;
	color:gray;
}
QPushButton:pressed{
	
	
	background-color: rgb(188, 0, 0);
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Random</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_5">
       <widget class="QPushButton" name="ledsProgramFlashButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>124</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift Light</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	
	background-color: rgb(49, 49, 49);
	border: 2px solid rgb(181, 0, 0);
	border-radius:20px;
	color:gray;
}
QPushButton:pressed{
	
	
	background-color: rgb(188, 0, 0);
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Flash</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_4">
       <widget class="QPushButton" name="ledsProgramPoliceButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>124</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift Light</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	
	background-color: rgb(49, 49, 49);
	border: 2px solid rgb(181, 0, 0);
	border-radius:20px;
	color:gray;
}
QPushButton:pressed{
	
	
	background-color: rgb(188, 0, 0);
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Police</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameLedControls">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>350</width>
      <height>200</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frameLedLabels">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>RED</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>GREEN</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>BLUE</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="styleSheet">
        <string notr="true">QSlider::groove:horizontal{
	background:transparent;
	
}
QSlider::handle:horizontal{
	border-radius:5px;
	width:22px;
	
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>23</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="ledsSliderRed">
          <property name="styleSheet">
           <string notr="true">QSlider::handle:horizontal{
	background:rgb(240, 0, 0 );
	
}
QSlider::sub-page:horizontal{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(0, 0, 0, 0), stop:1 rgb(255, 0, 0 ));
}</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="ledsSliderGreen">
          <property name="styleSheet">
           <string notr="true">QSlider::handle:horizontal{
	background:rgb(0, 240, 0 );
	
}
QSlider::sub-page:horizontal{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(0, 0, 0, 0), stop:1 rgb(0, 255, 0 ));
}</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="ledsSliderBlue">
          <property name="styleSheet">
           <string notr="true">QSlider::handle:horizontal{
	background:rgb(0, 0, 240);
	
}
QSlider::sub-page:horizontal{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(0, 0, 0, 0), stop:1 rgb(0, 0, 255));
}</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frameLedValues">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="ledsValueRed">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>65</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift Light</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ledsValueGreen">
          <property name="font">
           <font>
            <family>Bahnschrift Light</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ledsValueBlue">
          <property name="font">
           <font>
            <family>Bahnschrift Light</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameLedColorPicker">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>70</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight SemiConde</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>Color Picker</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ledsColorPickerButton">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/color_icons/Resources/Icons/color_icons/color_gray.png</normaloff>
         <activeoff>:/color_icons/Resources/Icons/color_icons/color_lightgray.png</activeoff>:/color_icons/Resources/Icons/color_icons/color_gray.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>240</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
